terraform/modules/gost_api/task.tf

```diff
@@ -233,12 +233,17 @@ resource "aws_iam_role" "task" {
 }
 
 resource "aws_iam_role_policy" "task" {
-  for_each = !var.enabled ? {} : {
-    connect-to-postgres   = module.connect_to_postgres_policy.json
-    ecs-exec              = module.ecs_exec_policy.json
-    send-emails           = module.send_emails_policy.json
-    rw-arpa-audit-reports = module.access_arpa_reports_bucket_policy.json
-  }
+  for_each = !var.enabled ? {} : merge(
+    {
+      connect-to-postgres   = module.connect_to_postgres_policy.json
+      ecs-exec              = module.ecs_exec_policy.json
+      send-emails           = module.send_emails_policy.json
+      rw-arpa-audit-reports = module.access_arpa_reports_bucket_policy.json
+    },
+    length(var.data_migration_bucket_names) == 0 ? {} : {
+      rw-data-migration-buckets = module.data_migration_bucket_names[0].json
```

Comment:
```suggestion
      rw-data-migration-buckets = var.data_migration_bucket_names[0].json
```

Reply:
Looks like I was mistakenly referencing the name of the variable and not the name of the module providing the JSON policy document. Should be fixed now.